### DISCUSSION TITLE
How to generate Pydantic class definition

### DISCUSSION THREAD

**Discussion by zys864** [tiangolo/pydantic-sqlalchemy#87] [Questions] [answered]
How to generate Pydantic class definition?
so we can copy it,then we can manual edit genrated pydantic class

**Comment by tiangolo** [comment, marked as answer]
Thanks for the patience with my reply! :sweat_smile:

I deprecated this library a while ago in favor of SQLModel which would solve all the same use cases in a much better way, and I'm directing all the effort in that direction.

SQLModel uses Pydantic models to define both data and table (SQL) models, so it's easier to do all these things. :rocket: 

And with SQLModel there's no need for that because you start with a simpler model that is both a SQLAlchemy model and a Pydantic model. :sunglasses:
